server.py

```diff
@@ -345,13 +348,9 @@ def server_points(server):
 			points += min(4, server["pop_v"] / 2)
 
 			# -8 for unrealistic max_clients
-			if server["clients_max"] > 200:
+			if server["clients_max"] > 500:
```

Comment:
No server can handle this amount of players. Even 200 is quite unrealistic.

Reply:
This is all theory.  But it should not be at all.  This number is just a reasonable measure to not get too many players and not to make it damn slow.  After some mods were rewritten, 100 people did not become a problem for me.  What happens if 150 people connect to me?  200?  Why exactly this number?  121 players did not break the server